doc/source/api/core/meshing/index.rst

```diff
@@ -103,10 +103,10 @@ TUI commands example
 .. code:: python
 
     import ansys.fluent.core as pyfluent
-    meshing_session = pyfluent.launch_fluent(mode="meshing")
-    meshing_session.tui.file.read_case("elbow.cas.gz")
-    solver_session = meshing_session.switch_to_solver()
-    solver_session.tui.define.models.unsteady_2nd_order("yes")
+    _meshing = pyfluent.launch_fluent(mode="meshing")
```

Comment:
Any particular reason to use the _leading underscore_ idiom here?  Not against it, just wondering...

Reply:
Only to avoid a name clash a little later. I will rename it anyway (this is draft but thanks for highlighting it).